config/helm/prometheus-operator/templates/prometheus-operator/clusterrole.yaml

```diff
@@ -49,6 +49,7 @@ rules:
   - pods
   verbs:
   - list
+  - get
```

Comment:
does the prometheus operator need this?

Reply:
I don't think so. Just added it as it already had 'list' and I could re-trigger test here (they were succeeding locally). I'll remove it